main.py

```diff
@@ -1,23 +1,21 @@
 import argparse
+import os
```

Comment:
Most of these changes are not related to the video mediator functionality, its just cleaning up this code/making it operable